packages/backend/src/services/CatalogService/CatalogService.ts

```diff
@@ -535,14 +518,41 @@ export class CatalogService<
                 analytics: await this.getFieldAnalytics(
                     user,
                     projectUuid,
-                    metric.tableName,
-                    metric.name,
+                    getItemId({
+                        name: metric.name,
+                        table: metric.tableName,
+                    }),
                 ),
             }));
 
         return {
             pagination,
-            data: await Promise.all(analyticsPromises),
+            data: await Promise.all(analyticsPromises), // ! This is very bad practive, temporary until we have `chart_usage` count
```

Comment:
do you think we can remove this?

Reply:
And wait until we add differently ? I want to basically check how the scheduled task does in analytics for example